app.js

```diff
@@ -1,21 +1,29 @@
 let express = require('express')
```

Comment:
Yea I agree with this! Immutability! Reading more about it I think it's better to use const over let